Gateway/Command/CreditCardStrategyCommand.php

```diff
@@ -109,7 +109,7 @@ public function execute(array $commandSubject)
                 $payment->addTransactionCommentsToOrder(
                     $payment->addTransaction(Transaction::TYPE_CAPTURE, $invoice),
                     __(
-                        'Captured amount of %1 online via Opn Payments Gateway.',
+                        'Captured amount of %1 online via Omise Payments Gateway.',
```

Comment:
```suggestion
                        'Captured amount of %1 online via Omise Payment Gateway.',
```

Same comment on other places